meorg_client/client.py

```diff
@@ -95,34 +115,32 @@ def login(self, email: str, password: str):
         Raises
         ------
         Exception
-            When the login fails. 
+            When the login fails.
         """
 
         # Assemble payload
         login_data = {
-            'email': email,
-            'password': hl.sha256(password.encode('UTF-8')).hexdigest(),
-            'hashed': 'true'
+            "email": email,
+            "password": hl.sha256(password.encode("UTF-8")).hexdigest(),
+            "hashed": "true",
         }
 
         # Call
-        response = self._make_request('post', endpoint='login', data=login_data)
+        response = self._make_request("post", endpoint="login", data=login_data)
 
```

Comment:
get 'post' from const and create url const for endpoint

Reply:
All urls moved to constants.py - see previous comment regarding method constants.